src/foam/u2/filter/FilterView.js

```diff
@@ -473,7 +474,59 @@ foam.CLASS({
             continue;
         }
       }
+      this.mementoUpdated = true;
+      this.mementoToggle = ! this.mementoToggle;
       return counter;
+    },
+    async function updateFilters() {
+      var of = this.dao && this.dao.of;
+
+      if ( ! of ) this.filters = [];
+      
+      var searchColumns_ = await this.filterPropertiesByReadPermission(this.searchColumns, of.id);
+      if ( searchColumns_ && searchColumns_.length > 0 ) {
+        this.filters =  searchColumns_;
+        return;
+      }
+
+      var columns = of.getAxiomByName('searchColumns');
+      columns = columns && columns.columns;
+      columns = await this.filterPropertiesByReadPermission(columns, of.id);
+      if ( columns ) {
+        this.filters = columns;
+        return;
+      }
+
+      columns = of.getAxiomByName('tableColumns');
+      columns = columns && columns.columns;
+      columns = await this.filterPropertiesByReadPermission(columns, of.id);
+      if ( columns ) {
+        this.columns = columns.filter(function(c) {
+        //  to account for nested columns like approver.legalName
+        if ( c.split('.').length > 1 ) return false;
+
+        var a = of.getAxiomByName(c);
+
+        if ( ! a ) console.warn("Column does not exist for " + of.name + ": " + c);
+
+        return a
+          && ! a.storageTransient
```

Comment:
why filter out 'storageTransient'?  storageTransient may not be persisted to disk but can be indexed in MDAO and are marshalled by medusa.

Reply:
This is old logic, but I'm assuming it was added for consistency purposes. If the data is not actually stored server-side, we don't want it to be displayed in any way, even if it is stored in the MDAO.